precompiles/gov/gov.go

```diff
@@ -0,0 +1,130 @@
+package gov
+
+import (
+	"bytes"
+	"embed"
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+	pcommon "github.com/sei-protocol/sei-chain/precompiles/common"
+)
+
+const (
+	VoteMethod    = "vote"
+	DepositMethod = "deposit"
+)
+
+const (
+	GovAddress = "0x0000000000000000000000000000000000001006"
```

Comment:
nit - is there a way to ensure that we don't reuse an addr for precompile? maybe we should add some precompile registry (or something like in app.go) and have a unit test

Reply:
yeah i can add a check in the setup code